tests/tests.cpp

```diff
@@ -30,7 +35,7 @@ class APITest : public ::testing::Test
 // Test Controller endpoint
 TEST_F(APITest, GetController)
 {
-    auto response = cpr::Get(cpr::Url{BASE_URL + "/controller"});
+    auto response = cpr::Get(cpr::Url{BASE_URL + "/controller"}, noPersistParam);
```

Comment:
Make sure changes made by the test are not persisted. Also applies to all the other tests.

Reply:
I got excited for a moment when I saw the macro, but then saw this was in tests, so it's OK ;-)